### ISSUE TITLE
Add an option to make tooltips hoverable

### ISSUE THREAD

**Issue by domoritz** [vega/vega-tooltip#890]
Tooltips disappear as soon as the cursor exits the mark. However, sometimes you may want it to stay while the user is hovering over the tooltip so that they can look at details/scroll/copy etc. It would be nice to have an option to enable this feature.

**Comment by Cyrik** [comment]
Agreed. I was looking for the same thing to make the tooltip scrollable.

**Comment by hydrosquall** [comment]
One way I've seen folks approach this is with a "cursor lock" (i.e. press a hotkey to pin the tooltip, regardless of where your mouse is). This is helpful for taking screenshots where you want the tooltip visible, or need the tooltip to be interacted with (to copy text, expand a collapsed section, etc)
